src/fswAlgorithms/attGuidance/prescribedRot2DOF/_UnitTest/test_prescribedRot2DOF.py

```diff
@@ -0,0 +1,321 @@
+    prescribed body's MRP attitude converges to both reference attitudes for a series of initial and reference attitudes
+    and maximum angular accelerations. (``sigma_FM_Final1`` is checked to converge to ``sigma_FM_Ref1``, and
+    ``sigma_FM_Final2`` is checked to converge to ``sigma_FM_Ref2``). Additionally, the prescribed body's final angular
+    velocity magnitude ``thetaDot_Final`` is checked for convergence to the reference angular velocity magnitude,
+    ``thetaDot_Ref``.
+
+    **Test Parameters**
+
+    Args:
+        thetaInit (float): [rad] Initial PRV angle of the F frame with respect to the M frame
+        thetaRef1a (float): [rad] First reference PRV angle for the first attitude maneuver
+        thetaRef2a (float): [rad] Second reference PRV angle for the first attitude maneuver
+        thetaRef1b (float): [rad] First reference PRV angle for the second attitude maneuver
+        thetaRef2b (float): [rad] Second reference PRV angle for the second attitude maneuver
+        phiDDotMax (float): [rad/s^2] Maximum angular acceleration for the attitude maneuver
+        accuracy (float): absolute accuracy value used in the validation tests
+
+    **Description of Variables Being Tested**
+
+    The prescribed body's MRP attitude at the end of the first maneuver ``sigma_FM_Final1`` is checked to converge to
+    the first reference attitude ``sigma_FM_Ref1``. The prescribed body's MRP attitude at the end of the second
+    maneuver ``sigma_FM_Final2`` is checked to converge to the second reference attitude ``sigma_FM_Ref2``.
+    Additionally, the prescribed body's final angular velocity magnitude ``thetaDot_Final`` is checked for convergence
+    to the reference angular velocity magnitude, ``thetaDot_Ref``.
+    """
+
+    [testResults, testMessage] = PrescribedRot2DOFTestFunction(show_plots, thetaInit, thetaRef1a, thetaRef2a, thetaRef1b, thetaRef2b, phiDDotMax, accuracy)
+
+    assert testResults < 1, testMessage
+
+
+def PrescribedRot2DOFTestFunction(show_plots, thetaInit, thetaRef1a, thetaRef2a, thetaRef1b, thetaRef2b, phiDDotMax, accuracy):
+    """Call this routine directly to run the unit test."""
+    testFailCount = 0                                        # zero unit test result counter
+    testMessages = []                                        # create empty array to store test log messages
+    unitTaskName = "unitTask"
+    unitProcessName = "TestProcess"
+    bskLogging.setDefaultLogLevel(bskLogging.BSK_WARNING)
+
+    # Create a sim module as an empty container
+    unitTestSim = SimulationBaseClass.SimBaseClass()
+
+    # Create the test thread
+    testProcessRate = macros.sec2nano(0.5)     # update process rate update time
+    testProc = unitTestSim.CreateNewProcess(unitProcessName)
+    testProc.addTask(unitTestSim.CreateNewTask(unitTaskName, testProcessRate))
+
+    # Create an instance of the =module that is being tested
+    PrescribedRot2DOFConfig = prescribedRot2DOF.PrescribedRot2DOFConfig()
```

Comment:
:small_red_triangle_down: Variable `PrescribedRot2DOFConfig` should be camelCase. So `prescribedRot2DOFConfig`.  Same for `HingedRigidBodyMessageData1` and `HingedRigidBodyMessageData2`

Reply:
Done.